### PR TITLE
mbt-87 add unit testing from node 12.22 to node 20.1

### PR DESCRIPTION
**Pull request by yutaoj**
Before submitting a pull request, please ensure the following:

## Description

Please explain the changes you made.

Add a link to the design (if applicable).

### Checklist
- [ ] Code compiles correctly
- [ ] Relevant tests were added (unit / contract / integration)
- [ ] Relevant logs were added
- [ ] Formatting and linting run locally successfully
- [ ] All tests pass
- [ ] UA review
- [ ] Design is documented
- [ ] Extended the README / documentation, if necessary
- [ ] Open source is approved

### CODE REVIEW

**Review by wangshen-sap — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -603,6 +685,42 @@ workflows:
           filters:
             branches:
               only: master
+      - build-node20:
```
Comment (young-yang03) [inline .circleci/config.yml:688]: Pls add new build nodes to workflow "on_pr_build"

File: .circleci/config.yml [hunk 2]
```diff
@@ -581,6 +663,42 @@ workflows:
...
+              ignore: master
+      - build-node14:
+          filters:
+            branches:
+              ignore: master
+      - build-node13-14:
+          filters:
+            branches:
+              ignore: master
+      - build-node12-22:
```
Comment (young-yang03) [inline .circleci/config.yml:698]: Add build-node** nodes to require part of test node

File: .circleci/config.yml [hunk 3]
```diff
@@ -603,6 +721,42 @@ workflows:
...
```
Comment (young-yang03) [inline .circleci/config.yml:756]: Add build-node** nodes to require part of test node

**Review by young-yang03 — APPROVED** [review]

File: .circleci/config.yml [hunk 4]
```diff
@@ -603,6 +721,42 @@ workflows:
...
             branches:
               only: master
+      - build-node20:
+          filters:
+            branches:
+              ignore: master
+      - build-node19:
+          filters:
+            branches:
+              ignore: master
```
Comment (young-yang03) [inline .circleci/config.yml:731]: ignore -> only

**Pull request merged**